pyshelf/configure.py

```diff
@@ -28,3 +29,10 @@ def logger(logger, log_level_name):
     handler.addFilter(RequestLogFilter())
     logger.addHandler(handler)
     logger.setLevel(log_level)
+
+
+def configure_app(app):
```

Comment:
eh, yeah, can we rename this to "app" seems weird but reads well at the module level

```
import configure
configure.app(app)
```